crates/holochain/tests/tests/websocket/mod.rs

```diff
@@ -490,15 +490,11 @@ async fn conductor_admin_interface_runs_from_config() -> Result<()> {
     let (client, rx) = websocket_client(&conductor_handle).await?;
     let _rx = WsPollRecv::new::<AdminResponse>(rx);
 
-    let dna = fake_dna_zomes("", vec![(TestWasm::Foo.into(), TestWasm::Foo.into())]);
-    let (fake_dna_path, _tmpdir) = write_fake_dna_file(dna).await.unwrap();
-    let register_payload = RegisterDnaPayload {
-        modifiers: DnaModifiersOpt::none(),
-        source: DnaSource::Path(fake_dna_path),
+    let request = AdminRequest::ListApps {
```

Comment:
As far as I can tell, the type of admin call is irrelevant here so I switched to using `ListApps`.